aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -4,12 +4,13 @@ use Sys::Hostname;
 use CAF::FileWriter;
 use CAF::Object;
 use NCM::Component::ks qw (ksuserhooks);
+use AII::Shellfe qw(:pxe_constants);
```

Comment:
why are these defined in `AII::Shellfe`? there are not used there at all (at first glance)